### PR TITLE
[16.0][ADD] product_english_name module

### PR DESCRIPTION
**Pull request by bealdav**
screen_french

cc @legalsylvain @HaraldPanten @rousseldenis @Kev-Roche

### CODE REVIEW

**Review by PaulGoubert — APPROVED** [review]
Functionnal review OK

File: product_english_name/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- active another language than english and set it to your user.
```
Comment (bosd) [inline product_english_name/readme/CONFIGURE.rst]:
```suggestion
- activate another language than English and set it as your user language.
```

File: product_english_name/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- add your user to 'Show english name' group.
+- browse your products in form view.
```
Comment (bosd) [inline product_english_name/readme/CONFIGURE.rst]:
```suggestion
- browse your products in a form view.
```

File: product_english_name/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+This module display product name in english near name in user language.
+
+Use case:
+Product name is a very important information at least you buy products in a foreign country.
```
Comment (bosd) [inline product_english_name/readme/DESCRIPTION.rst]:
```suggestion
The Product name is very important information at least if you buy products in a foreign country.
```

File: product_english_name/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+It often occurs that in an odoo instance product names don't match with their translations.
+This module can help to detect error on translations.
```
Comment (bosd) [inline product_english_name/readme/DESCRIPTION.rst]:
```suggestion
This module can help to detect those translation errors.
```

File: product_english_name/views/product_template.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_form_view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//h1[1]" position="after">
+                <h5 groups="product_english_name.show_english_name">
```
Comment (bosd) [inline product_english_name/views/product_template.xml:9]:
```

2024-09-30 14:58:43,884 478 WARNING odoo odoo.addons.base.models.ir_ui_view: The group 'product_english_name.show_english_name' defined in view does not exist!
View error context:
{'file': '/__w/product-attribute/product-attribute/product_english_name/views/product_template.xml',
 'line': 3,
 'name': 'product.template form',
 'view': ir.ui.view(1841,),
 'view.model': 'product.template',
 'view.parent': ir.ui.view(464,),
 'xmlid': 'product_template_form_view'} 
```

File: product_english_name/readme/DESCRIPTION.rst [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+.. figure:: ../static/description/screen_french.png
+    :alt: Display product name in english near name in user language
```
Comment (bosd) [inline product_english_name/readme/DESCRIPTION.rst]:
```suggestion
    :alt: Display the product name in English near the product name in users language
```

File: product_english_name/security/groups.xml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<odoo>
+
+    <record id="show_english_name" model="res.groups">
+        <field name="name">Show english name</field>
```
Comment (bosd) [inline product_english_name/security/groups.xml]:
```suggestion
        <field name="name">Show English name</field>
```

File: product_english_name/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Akretion
+{
+    "name": "Product English Name",
+    "summary": """Display product name in english """,
```
Comment (bosd) [inline product_english_name/__manifest__.py]:
```suggestion
    "summary": """Display the product name in English besides the product name in the users language. """,
```

File: product_english_name/readme/DESCRIPTION.rst [hunk 9]
```diff
@@ -0,0 +1,12 @@
+This module display product name in english near name in user language.
```
Comment (bosd) [inline product_english_name/readme/DESCRIPTION.rst]:
```suggestion
This module displays the product name in English near the products name in user's language.
```

File: product_english_name/readme/DESCRIPTION.rst [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+
+Use case:
+Product name is a very important information at least you buy products in a foreign country.
+It often occurs that in an odoo instance product names don't match with their translations.
```
Comment (bosd) [inline product_english_name/readme/DESCRIPTION.rst]:
```suggestion
It often occurs that in an odoo instance the product names don't match with their translations.
```

**Review by bosd — CHANGES_REQUESTED** [review]
Thanks for this attribution.
Some minor comments about warning and language.

Adittionally non blocking comment.
Compatability with product profile could be improved. As seen on runboat:
image

**Comment by bealdav** [comment]
Thanks a lot, @bosd for your review

**Comment by bealdav** [comment]
image

Finally works

**Review by Kev-Roche — APPROVED** [review]
LGTM

**Review by mourad-ehm — APPROVED** [review]
Code review

**Comment by bealdav** [comment]
Hi @bosd could you check again ?

**Review by bosd — APPROVED** [review]

**Comment by HaraldPanten** [comment]
@bealdav You should squash your commits.

**Comment by bealdav** [comment]
@HaraldPanten thanks to notified me. Fixed

**Comment by HaraldPanten** [comment]
Hi, the commit name should appear without the version number, something like:

[ADD] product_english_name module: description in few words

**Comment by bealdav** [comment]
@HaraldPanten where this rule is described ?

**Comment by pedrobaeza** [comment]
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message

**Comment by bealdav** [comment]
I don't see in the doc, that it's required to not add version for a new module.

But I'm ok to change my commit.

**Comment by HaraldPanten** [comment]
> I don't see in the doc, that it's required to not add version for a new module.
> 
> But I'm ok to change my commit.

Hi @bealdav We have to follow Odoo's tags, and then apply OCA rules in case there are no examples ([MIG] is one of these use cases).

You can see that Odoo never uses the version number in the commit messages, as well --> https://github.com/odoo/odoo/pulls and here you can see Odoo's guidelines (for tagging) --> https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html#tag-and-module-name

**Comment by bealdav** [comment]
That's what I have applied, we're OK then
`[ADD] for adding new modules`

**Comment by bealdav** [comment]
That seems to be nitpicking, but I'm OK to change my commit as said previously.
> You can see that Odoo never uses the version number in the commit messages

**Comment by bealdav** [comment]
Done, thanks for reviews

**Review by PaulGoubert — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Didn't test the module in detail, but it has several reviews from other users, so... I'll try to help for moving forward.

THX for contributing!

**Pull request merged**